modules/navier_stokes/src/userobjects/CNSFVMinmaxSlopeLimiting.C

```diff
@@ -73,9 +74,11 @@ CNSFVMinmaxSlopeLimiting::limitElementSlope() const
 
   for (unsigned int is = 0; is < nside; is++)
   {
-    if (elem->neighbor(is) != NULL)
+    neighbor = elem->neighbor_ptr(is);
```

Comment:
Go ahead and declare the variable here. There's no need to declare it at the top of the method. Keep the scope as close to the usage as possible.